src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/ModelRunController.kt

```diff
@@ -12,19 +13,26 @@ class ModelRunController(val fileManager: FileManager, val apiClient: APIClient)
     @PostMapping("/run/")
     @ResponseBody
     fun run(@RequestBody options: Map<String, Any>): ResponseEntity<String> {
-        val allFiles = fileManager.getAllFiles()
+        val allFiles = fileManager.getAllHashes()
```

Comment:
In fact this will probably change to use the new `getAllFiles` and we will deprecate this `getAllHashes` method, but waiting on a hintr change for this.